### ISSUE TITLE
Automatically switch between themes at scheduled times.

### ISSUE THREAD

**Issue by gravelbar** [Vencord/plugin-requests#772]
### Plugin Description

A plugin that allows you to switch between themes at scheduled times would be amazing. This would be useful for people who like light mode during the day and dark mode at night. Currently, I manually switch my themes at sunrise and sun fall, it would be nice for this to be automated.

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [ ] I checked every box, despite not reading any
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by radishcatish** [comment]
yeah, smth like how mobile iOS discord does it if you have discord and phone set to automatic theming (goes dark at night, light at day)

**Comment by maddie480** [comment]
Oh hey, I wanted the same thing and did an auto theme switcher plugin for myself on a Vencord fork: https://github.com/maddie480/Vencord/tree/feat/auto-theme-switcher
I wasn't sure whether to submit a PR or to turn it into a third-party plugin, though 😅
